x/dogfood/keeper/msg_server.go

```diff
@@ -104,5 +108,24 @@ func (k Keeper) UpdateParams(
 		nextParams.AssetIDs = prevParams.AssetIDs
 	}
 	k.SetParams(c, nextParams)
+
+	// update the related info in the AVS module
+	isAVS, avsAddr := k.avsKeeper.IsAVSByChainID(c, avstypes.ChainIDWithoutRevision(c.ChainID()))
+	if !isAVS {
+		return nil, errors.Wrapf(types.ErrNotAVSByChainID, "chainID:%s avsAddr:%s", c.ChainID(), avsAddr)
+	}
+	err := k.avsKeeper.UpdateAVSInfo(c, &avstypes.AVSRegisterOrDeregisterParams{
+		AvsName:           c.ChainID(),
+		AvsAddress:        avsAddr.String(),
+		AssetID:           nextParams.AssetIDs,
+		UnbondingPeriod:   uint64(nextParams.EpochsUntilUnbonded),
+		MinSelfDelegation: nextParams.MinSelfDelegation.Uint64(),
+		EpochIdentifier:   nextParams.EpochIdentifier,
+		ChainID:           c.ChainID(),
+		Action:            avskeeper.UpdateAction,
+	})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Given the multiple opportunities for this to error, should we use a wrapped context? Or are the errors solely cosmetic?

Reply:
fixed, it will return a wrapped error